jobs/status.go

```diff
@@ -68,9 +68,11 @@ func StartStatusReporter(ctx context.Context, scanCtx *scan.ScanContext) chan er
 	threshold := *opts.InactiveThreshold
 	go func() {
 		defer stopScanJob()
-		ticker := time.NewTicker(30 * time.Second)
+		ticker := time.NewTicker(1 * time.Second)
```

Comment:
Not sure we want to be that quick about cancelling, keeping 30 seconds would simplify the following logic

Reply:
I increased the granularity to 1 second as this gives us a chance to check the "Stopped" flag more frequently
Else we will be checking this flag once per 30 seconds .

Reply:
Yup, my point is I don't see any problem with use checking once per 30 seconds?
It simplifies the code and free some CPU usage